frontend/src/pages/home.tsx

```diff
@@ -151,9 +161,16 @@ const Home: FC = () => {
                         <>
                           <Divider className="my-8" />
                           <div className="text-right">
+                            {button.url && (
                             <Button component={Link} href={button.url} size="large">
                               {button.text}
                             </Button>
+                            )}
+                            {(button.url == null || button.url == undefined) && (
```

Comment:
This sound problematic because my assumption is that if there is no url for that tab panel's button that then it is always the quiz button.

Reply:
As per conversation, I added a note about further work required and where the work had been left. This is a FIXME technical debt, but the ticket isn't closed yet. So when new content/layout arrives we can revisit it then with said note in mind.